src/proxy.ts

```diff
@@ -0,0 +1,23 @@
+import * as debug from 'debug';
+
+const d = debug('@electron/got:proxy');
```

Comment:
`got`?

Reply:
I've been looking too much at `got` to figure out the proxy stuff :dizzy_face: